uk.co.compucorp.civicrm.hrleaveandabsences/views/shared/directives/leave-request-popup.html

```diff
@@ -65,7 +65,9 @@ <h4 class="modal-title">
               <crm-loading show="!$ctrl.loading.showBalanceChange">
                 <div ng-show="$ctrl.uiOptions.showBalance">
                   <hr/>
-                  <div class="row chr_leave-request-modal__form-group">
+                  <div
+                    class="row chr_leave-request-modal__form-group"
+                    ng-if="$ctrl.isRole('manager') || $ctrl.isLeaveStatus($root.sharedSettings.statusNames.awaitingApproval) || $ctrl.isLeaveStatus($root.sharedSettings.statusNames.moreInformationRequired)">
```

Comment:
I dont think its a good idea because then it will be exclusive for OR and to use with AND we need to call it separately, what we have now is better in my view